pkg/lib/machine-dialogs.js

```diff
@@ -428,26 +480,36 @@ function AddMachine(dialog) {
         if (!limit || isNaN(limit))
             limit = 20;
 
-        dialog.render({
-            nearlimit : limit * 0.75 <= dialog.machines_ins.list.length,
-            limit : limit,
-            placeholder : _("Enter IP address or host name"),
-            options : invisible,
-        });
+        self.user = { };
 
-        var button = dialog.get_sel(".modal-footer>.pf-m-primary");
-        button.on("click", add_machine);
+        cockpit.user()
+                .done(function (user) {
+                    self.user = user;
+                })
+                .always(function (user) {
+                    dialog.render({
+                        nearlimit : limit * 0.75 <= dialog.machines_ins.list.length,
+                        limit : limit,
+                        placeholder : _("Enter IP address or host name"),
```

Comment:
This input is focused by default, but all the other dialogs focus on the close button by default. This is a little inconsistent from a keyboard accessibility point of view.

Reply:
I took out the explicit focusing of the address input. I don't see any focus of any Close button, however, in any dialog that I have tried.